specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,125 @@
+`Microsoft.Windows.ApplicationModel.Resources` MRTCore namespace. This property
+(`Microsoft.Windows.ApplicationModel.Resources.PrimaryLanguageOverride`) overrides the behavior of
+the existing one (`Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride`).
+
+# API Pages
+
+## ApplicationLanguages class
+
+Specifies the language-related preferences that the app can use and maintain.
+
+# Remarks
+
+The languages referenced in this class are represented by
+[BCP-47](https://tools.ietf.org/html/bcp47) language tags. Each of these is a language identifier
+and not a locale name, although the two structurally can be the same. As a language tag, "en-US"
+means American English (or the dialect of English spoken in the United States). It makes no claims
+about the actual location of the speakers (though many of them do reside in the US) and no other
+claims about conventions that are used in the US such as the measurement system or currency.
+
+Example:
+
+```c#
+Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride = "en-US";
+```
+
+Language tags support the Unicode extensions "ca-" and "nu-". (See Unicode Key/Type Definitions.)
+Note that these extensions can affect the numeral system or calendar used by globalization objects.
+
+## ApplicationLanguages.PrimaryLanguageOverride property
+
+Gets or sets an override for the app's preferred language, expressed as a
+[BCP-47 language tag](https://tools.ietf.org/html/bcp47). Unlike
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride),
+this setting is global for the running process and is not persisted.
+
+```c#
+public static string PrimaryLanguageOverride { get; set; }
+```
+
+# Property Value
+
+[String](https://learn.microsoft.com/dotnet/api/system.string)
+
+A [BCP-47 language tag](https://tools.ietf.org/html/bcp47). The app can set it to override the
+language. It must be a single language tag; a delimited list of language tags will result in default
+language being used.
+
+The PrimaryLanguageOverride property should only be set to languages available for the app.
+
+When your app gets the value, PrimaryLanguageOverride returns either a single language tag (if your
+app has previously set the property) or an empty string.
+
+# Remarks
+
+Apps normally run with language settings determined by the system by comparing the languages
+supported by the app with the language preferences of the user. The PrimaryLanguageOverride property
+is used to override that behavior by setting a single specific language.
+
+The PrimaryLanguageOverride setting is not persisted between sessions. It should be set each time
+the app is loaded. You should set PrimaryLanguageOverride setting in early stage of app loading,
+before any resource is loaded. It should only be set based on user input presented in settings UI.
```

Comment:
is it possible to set early enough so that WinUI will be able to respect it?

Reply:
Setting it before InitializeComponent in App() constructor like presented here https://github.com/microsoft/WindowsAppSDK-Samples/pull/358/files#diff-af77dcd97ddae80357680b47edbe8940ebd597b31bf81dd152af69067e46f701R40
worked for me in every scenario I tested